app/app.js

```diff
@@ -1,14 +1,23 @@
 import Ember from 'ember';
-import Resolver from 'ember/resolver';
+import Resolver from './resolver';
 import loadInitializers from 'ember/load-initializers';
+import config from './config/environment';
+
+let App;
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).